runatlantis.io/docs/using-atlantis.md

```diff
@@ -167,6 +167,49 @@ atlantis import -d dir 'aws_instance.example["foo"]' i-1234567890abcdef0 -- -var
 ```
 If a flag is needed to be always appended, see [Custom Workflow Use Cases](custom-workflows.html#adding-extra-arguments-to-terraform-commands).
 
+---
+## atlantis state rm
+```bash
+atlantis state [options] rm ADDRESS... -- [terraform import flags]
+```
+### Explanation
+Runs `terraform state rm` that matches the directory/project/workspace.
+This command discards the terraform plan result. After an import and before an apply, another `atlantis plan` must be run again.
+
+### Examples
+```bash
+# Runs state rm
+atlantis state rm ADDRESS1 ADDRESS2
```

Comment:
Does this work with square brackets and single quotes? If so, we should show this in this list of examples.

Reply:
Oh nvm, I see it below.

Reply:
Oh nice and this is for multiple addresses. Cool!